src/main/java/com/tabnine/inline/TabnineDocumentListener.java

```diff
@@ -73,27 +69,7 @@ private boolean shouldIgnoreChange(DocumentEvent event, Editor editor, int offse
       return true;
     }
 
-    if (!CompletionUtils.isValidMidlinePosition(document, offset)) {
-      return true;
-    }
-
-    return isInTheMiddleOfWord(document, offset);
```

Comment:
this check is no longer necessary since the isValidMidlinePosition already checks this case